### PR TITLE
Only format /dev/sda if we are deploying to DigitalOcean

### PR DESCRIPTION
**Pull request by michaelfig**
This fixes an overzealous `ag-setup-cosmos` script that would reformat and mount `/dev/sda` even if not on a terraformed DigitalOcean instance as intended.

The surprise was fortunately not fatal to @dckc's Docker host but was unnecessarily nerve-wracking.

Also tweak `ag-setup-cosmos provision` to properly update terraform and clear/rebuild derived state.

### CODE REVIEW

File: packages/deployment/ansible/roles/prereq/tasks/main.yml [hunk 1]
```diff
@@ -20,25 +20,30 @@
     - xfsprogs
     - "nodejs={{ NODEJS_VERSION }}.*"
 
-- name: Stat /dev/sda
+- name: Default home_dev for DigitalOcean
+  set_fact:
+    home_dev: /dev/sda
+  when: "home_dev is not defined and provider is defined and provider.startswith('digitalocean')"
```
Comment (dckc) [inline packages/deployment/ansible/roles/prereq/tasks/main.yml:26]: `is defined`? what language is that? oh... TIL: [Conditionals — Ansible Documentation](https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html)

File: packages/deployment/src/main.js [hunk 2]
```diff
@@ -918,11 +918,12 @@ ${name}:
             // These are the validator params.
             roleParams = `
   moniker: Agoric${offset + instance}
-  website: https://testnet.agoric.com
+  website: https://testnet.agoric.net
```
Comment (dckc) [inline packages/deployment/src/main.js:925]: drive-by?

**Review by dckc — APPROVED** [review]
good stuff.

comments are minor

File: packages/deployment/src/main.js (hunk 2 above)
Comment (michaelfig) [reply packages/deployment/src/main.js:925]: > drive-by?

Yep, couldn't leave agoric.com alone.
